src/utils/drawing.js

```diff
@@ -112,6 +121,72 @@ export function drawCropmarks(papersize, per_sheet) {
   return lines;
 }
 
+/**
+ * @param {@param {import("../book.js").PageInfo}} sigDetails - information about signature where marks will be printed
+ * @param {import("../book.js").Position} position - position info object
+ * @param {number} amount - amount of sewing crosses.
+ * @param {number} marginPt - distance from the end of sheet of paper to kettle mark
+ * @param {number} tapeWidthPt - distance between two points in a single sewwing cross.
+ * @returns {Point[]}
+ */
+export function drawSewingMarks(sigDetails, position, amount, marginPt, tapeWidthPt) {
+  // Here normalize coordinates to always think in x an y like this
+  // | P        |H|    P |
+  // |  A       |E|   A  |
+  // |   G      |I|  G   |
+  // |    E     |G| E    |
+  // |          |T|      |
+  // |-POSITION-| |      |
+
+  // Left pages has spine position on the endge :/
```

Comment:
nice comment!

Reply:
```suggestion
  // Left pages have spine position on the edge :/
```

Reply:
Changed spelling :)

Reply:
OH!!! omg - I meant I was genuinely excited by the documentation about page information!! 
absolutely did not notice spelling/grammar!! gah- I would never call out such things, even if I did notice them!
so sorry - really - just was trying to express happiness at code improving

Reply:
No problem :)